src/main/java/com/solvd/elasticsearchmicroservice/kafka/KfConsumer.java

```diff
@@ -0,0 +1,9 @@
+package com.solvd.elasticsearchmicroservice.kafka;
+
+import com.solvd.elasticsearchmicroservice.kafka.event.NoteEvent;
+
+public interface KfConsumer {
```

Comment:
@vpetrovaa make it more generic, that Consumer can receive not only not events but other objects too

Reply:
@h1alexbel fixed https://github.com/vpetrovaa/elasticsearch-microservice/pull/4/commits/16f3b38bd82af60737119d28086db48a94d9489c